Library/ViewModels/PledgeCreditCardViewModel.swift

```diff
@@ -65,10 +65,18 @@ public final class PledgeCreditCardViewModel: PledgeCreditCardViewModelInputs,
     self.expirationDateText = creditCard
       .map { Strings.Credit_card_expiration(expiration_date: $0.expirationDate()) }
 
-    self.notifyDelegateOfCardSelected = creditCard
+    let creditCardInitial = creditCard
+      .map { $0.id }
+
+    let creditCardSelected = creditCard
       .takeWhen(self.selectButtonTappedProperty.signal)
       .map { $0.id }
 
+    self.notifyDelegateOfCardSelected = Signal.merge(
+      creditCardInitial,
```

Comment:
We now make sure to notify delegate when payment methods get loaded initially and default card gets selected (if there's at least one card).

Reply:
This is also fixed in https://github.com/kickstarter/ios-oss/pull/856

Reply:
Good to know...I can see it looks slightly different to my approach...I can either way for the other PR to make it in or try reflecting those changes here.